### PR TITLE
Replace `_.pick` with native JS

### PR DESCRIPTION
**Pull request by XhmikosR**
Refs #604

### CODE REVIEW

File: lib/svg-sprite/config.js [hunk 1]
```diff
@@ -145,7 +145,8 @@ function SVGSpriterConfig(config) {
             meta = meta ? yaml.load(meta) : {};
             for (const m in meta) {
                 if (_.isPlainObject(meta[m])) {
-                    this.shape.meta[path.join(path.dirname(m), path.basename(m, '.svg'))] = _.pick(meta[m], ['title', 'description']);
+                    const { title, description } = meta[m];
+                    this.shape.meta[path.join(path.dirname(m), path.basename(m, '.svg'))] = { title, description };
```
Comment (XhmikosR) [inline lib/svg-sprite/config.js:149]: Now that I see this, maybe we could skip the new var and directly access the keys. Let me know what you prefer

Comment (Kreeg) [reply lib/svg-sprite/config.js:149]: I think it's ok here. But I personally prefer skipping such variables. In this case I see no problem

**Comment by XhmikosR** [comment]
@Kreeg if you think any of the PRs are ready feel free to approve. Generally, in the lodash removal PRs, we need to make sure there's no TypeError in case something is undefined/null, but other than that it should be safe.

If you think a PR needs something else please try to push a patch :)

**Comment by Kreeg** [comment]
@XhmikosR Hi!
The problem here is that we can not be 100% sure about these changes since there is no unit tests at all.
IMHO 2.0 release should focus on unit test coverage. Only after it we can do such refactoring.

**Comment by XhmikosR** [comment]
No problem, I have already landed a few changes like this, but I don't mind waiting.

Regardless, this applies to all PRs, some might be safer to land. In these cases, feel free to approve/push further fixes :)

**Review by Kreeg — APPROVED** [review]

File: lib/svg-sprite/config.js (hunk 1 above)
Comment (XhmikosR) [reply lib/svg-sprite/config.js:149]: My only concern is if we end up with undefined `title` and/or `description`. If so, the new code will probably throw.

**Comment by XhmikosR** [comment]
@Kreeg I double checked this and it seems like we are safe, but I could use more eyes.

**Pull request merged**